CODE_OF_CONDUCT.md

```diff
@@ -294,7 +294,7 @@ involved:
 
 1.  **Correction**
     -   **Behavior:** Use of inappropriate language or other minor violations
-        the code of conduct.
+        of the code of conduct.
```

Comment:
This "of" needs to be on the previous line (we use prettier for auto-formatting):
https://github.com/carbon-language/carbon-lang/runs/7568551514?check_suite_focus=true

Reply:
Fixed